purchase_stock_picking_return_invoicing/models/purchase_order.py

```diff
@@ -127,15 +127,35 @@ def _compute_qty_refunded(self):
         'move_ids.returned_move_ids.state',
     )
     def _compute_qty_returned(self):
+        """Made through read_group for not impacting in performance."""
+        ProductUom = self.env['product.uom']
+        StockMove = self.env['stock.move']
```

Comment:
Use [snake_case for variables names](https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#variable-names) and [CamelCase for class names](https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#odoo-python-classes).

Reply:
This is a special case that has been discussed a lot of times. This represents something similar as a class in Odoo context, and it's used by Odoo itself this way.

Reply:
If you can a new special case you should added to our guidelines.
For now, our guidelines of the OCA (not odoo) is use CamelCase for class name and snake_case for variables, uppercase for global variable.
If odoo uses other standard that is the odoo standard.
If you want join the odoo and oca standard and avoid discuss too many times you need added to the guidelines document.

Reply:
It's not a special case, as this is not the variable mentioned in the guidelines, and the linter is not forcing us to do the way you want. Open if you want the discussion for forbidding it and then I won't use it if approved.

Reply:
Your first comment:
> This is a special case...

Your second comment:
> It's not a special case...

I think that you are contradicting the guidelines and your self comments.

But I don't have problem if you want skip our guideline [variable-names](https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#variable-names) for now. If in the future you decide added it to avoid discussed it again and again it is good for me too.

I just approved this one.
Thanks for the great work here.

Reply:
Well, I haven't used the proper words, but they were following your comments. Any way, what I meant to say basically is: "this one is out of the guidelines". Call is "special case" or not. Remember that I was answering your sentence:

> If you like a new special case you should added to our guidelines.

So the proper answer would be "this is not the kind of special case inside the rule, as this can't be considered a normal variable (and thus is a special variable case)". Better?

I don't want to skip things, simply that they are not covered, but you know me, we can discuss what you want for generalizing this rule, but I don't want to get blocked this PR for that reason, and this want blocks me a huge migration to the the performance question.